fava/templates/_aside.html

```diff
@@ -13,7 +13,11 @@
     {% if title_ %}<h3>{{ title_ }}</h3>{% endif %}
     <ul class="navigation">
     {% for id in menuitems %}
-        <li><a{% if id == active_page %} class="selected"{% endif %} data-key="{{ globals.all_pages[id].1 }}" href="{{ url_for('report', report_name=id) }}">{{ globals.all_pages[id].0 }}</a>
+        <li><a{% if id == active_page %} class="selected"{% endif %} data-key="{{ globals.all_pages[id].1 }}" href="{{ url_for('report', report_name=id) }}">{{ globals.all_pages[id].0 }}
+            {% if id == "events" and api.upcoming_events|length > 0 %}
```

Comment:
I'm sorry for this, but my python skills is almost zero... 
I understood other changes and please give me advise how do I must change this block?